Package.swift

```diff
@@ -67,7 +67,9 @@ let package = Package(
                 "ConfigurationTests/mock_fcm.json",
                 "ConfigurationTests/mock_invalid_fcm.json",
                 "ConfigurationTests/mock.p8",
-                "ConfigurationTests/mock.pem"
+                "ConfigurationTests/mock.pem",
+                "ConfiguartionTests/cert.pem",
+                "ConfiguartionTests/key.pem"
```

Comment:
Typo 🙃

Reply:
Thanks, I fixed this with https://github.com/Apodini/Apodini/commit/beff00cfd7e821767afd6ef7ec21e42903a4fdb6